pkgs/development/python-modules/nevow/default.nix

```
@@ -10,6 +10,14 @@ buildPythonPackage rec {
     sha256 = "2299a0d2a0c1312040705599d5d571acfea74df82b968c0b9264f6f45266cf6e";
   };
 
+  patches = [
+    # Fix builds against recent Twisted.
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/twisted/nevow/pull/109.patch";
```

please use the commit patch url
```suggestion
      url = "https://github.com/twisted/nevow/commit/f1b366f1a73009b6a1df12fa6f4dc464c564c944.patch";
```

---

Good point. Should be fixed now.